src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml

```diff
@@ -140,6 +140,25 @@ spec:
                         description: Generate indicates whether generate the assessment
                           report or not.
                         type: boolean
+                      governor:
+                        description: Indicate whether to config of governor
+                        properties:
+                          apiToken:
+                            description: Api token for user authentication
+                            type: string
```

Comment:
I think we have update governor properties at /src/tools/installation/yaml/crd.yaml
Can you check what is the impact when we don't add governor properties at crd.yaml

Reply:
Its working fine with governor properties.